owning-a-smart-contract.md

```diff
@@ -51,14 +51,14 @@ touch ./contracts/Restricted.aes
 
 The following code goes inside this file:
 
-```javascript=
+```aes
```

Comment:
```suggestion
```sophia
```

The name of the programming language is expected here not the file extension name. This would be `python` not `py` if this were to be a python code